installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -124,13 +124,14 @@ include::modules/oc-mirror-differential-updates.adoc[leveloffset=+2]
 // Performing a dry run
 include::modules/oc-mirror-dry-run.adoc[leveloffset=+1]
 
-// Mirroring Operator images in OCI format
+// Including local OCI Operator catalogs
 include::modules/oc-mirror-oci-format.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../operators/admin/olm-managing-custom-catalogs.adoc#olm-managing-custom-catalogs-fb[File-based catalogs]
+// TODO: This title might need to update per sebastian's PR
```

Comment:
If #58953 gets merged before this PR, then I'll need to update the title here. If not, then I'll ask @skopacz1 to update this title in his PR

Reply:
Circling back here to let you know that my PR has been merged.

Reply:
Thanks @skopacz1!